bootstrap/lib/cache.js

```diff
@@ -37,31 +50,58 @@ const withCache = (execute, options) => {
   // Algorithm we use to derive entry key
   const hashOptions = {
     algorithm: 'sha1',
+
     encoding: 'hex',
     excludeKeys: (key) => options.ignoredKeys.includes(key),
   }
+  const _getKey = (data) => hash(data, hashOptions)
+  const _getCoalescingKey = (key) => hash(`${options.requestCoalescingGroup}--${key}`, hashOptions)
```

Comment:
hashOptions seems like its not being used here, but i would question our usage of hashing the key here at all since its a simple string. Redis would handle this fine by itself and result in more debuggable logging if we ever have to look into Redis

Reply:
There are some extra bytes of memory that we are going to use if we go with the string key directly, but that is negligible for our use case. This is more to keep it consistent as the data key is also a hash.

I am going to leave this question open for now.

Reply:
@HenryNguyen5 improved this in [586e93](https://github.com/smartcontractkit/external-adapters-js/pull/94/commits/586e9377b7069d59ebd2fdacac2fc2f9c50b137a).